docs/iaas.md

```diff
@@ -98,3 +102,67 @@ http://localhost:30000/d/openstack-overview
+endpoint, there are some differences in Prometheus configuration and alerts.
+
+#### Prometheus metrics and alerts
+
+Ceph contains 2 build-in sources of metrics a.k.a. exporters.
+The Ceph exporter (introduced in Reef release of Ceph) is the main source of Ceph performance metrics. It runs as a
+dedicated daemon. This daemon runs on every Ceph cluster host and exposes a metrics end point where all the performance
+counters exposed by all the Ceph daemons running in the host are published in the form of Prometheus metrics.
+
+The second source of metrics is the Prometheus manager module. It exposes metrics related to the whole cluster,
+basically metrics that are not produced by individual Ceph daemons.
+
+Read the related Ceph [docs](https://docs.ceph.com/en/reef/monitoring/#ceph-metrics).
+Since these exporters are integrated with Ceph, deploying a third-party Ceph exporter is unnecessary.
+
+**Prometheus alerts**
+
+Both Ceph deployment strategies use the ceph-mixins project as a source of alerts. The ceph-ansible and rook projects
+each maintain a rendered version of these alerts, but the rook repository contains some differences, primarily because
+rook does not use the cephadm tool as a backend. 
+Therefore, find and apply one of the following commands to create a custom observer rules values file for either the
+ceph-ansible or ceph-rook deployment ([yq](https://github.com/mikefarah/yq/#install) tool required):
+
+```bash
+# ceph-ansible
+curl -s https://raw.githubusercontent.com/ceph/ceph/main/monitoring/ceph-mixin/prometheus_alerts.yml | \
+  yq '{"kube-prometheus-stack": {"additionalPrometheusRulesMap": {"ceph-rook-rules": (. + {"additionalLabels": {"prometheus_rule": "1"}})}}}' > iaas/values-observer-ceph-rules.yaml
+
+# rook
+curl -s https://raw.githubusercontent.com/rook/rook/master/deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml | \
+  yq '{"kube-prometheus-stack": {"additionalPrometheusRulesMap": {"ceph-rook-rules": (. + {"additionalLabels": {"prometheus_rule": "1"}})}}}' > iaas/values-observer-ceph-rules.yaml
+```
+
+#### Grafana dashboards
+
+We've tested and could recommend 2 sources of Grafana dashboards that are suitable for both Ceph deployment strategies (ansible and rook):
+- [dashboards linked in rook docs](https://rook.io/docs/rook/latest-release/Storage-Configuration/Monitoring/ceph-monitoring/?h=gra#grafana-dashboards)
+- [ceph-mixins dashboards](https://github.com/ceph/ceph-mixins/tree/master/dashboards)
+  - Built version of ceph-mixins dashboards could be found e.g. [here](https://github.com/ceph/ceph/tree/main/monitoring/ceph-mixin/dashboards_out)
+
+We consider the dashboards created within the Rook project as a solid starting point for Ceph metrics visualization.
+If you want to see more detailed dashboards, uncomment and use the ceph-mixin dashboards in the `values-observer-ceph-rook.yaml`
+or `values-observer-ceph-ansible.yaml` file. You can use both.
+
+#### Update the SCS monitoring deployment
```

Comment:
IMO this update will override `dashboardProviders` from `values-observer-iaas.yaml`. Is it possible to use together ceph and iaas dashboards?

Reply:
it should be possible to use them together, I will try to define another dashboard providers file for Grafana, e.g. `dashboardprovidersceph.yaml`

Reply:
Thanks.
Then, also `-f iaas/values-observer-iaas.yaml` can be added to the full command

Reply:
See 11379dcab2c27fb4f7890deb5905d6e32e737582

My intention was not to provide the full command, as IMO this might confuse the reader. IMO we would then need to mention that the prometheus-openstack-exporter should also be deployed etc. 

I would rather keep these things separated, as the reader can deploy both things (OpenStack related, Ceph related) 
step-by-step and then ends with both deployed.  An advanced user can apply the full command you mention, but I would like to keep it simple and bring this full command out-of-scope to this tutorial.

Reply:
Ok, I agree